wallet/src/routes.rs

```diff
@@ -770,6 +798,10 @@ pub async fn get_records(wallet: &mut Option<Wallet>) -> Result<Vec<RecordInfo>,
     Ok(wallet.records().await.collect::<Vec<_>>())
 }
 
+pub async fn get_last_keystore() -> Result<PathBuf, tide::Error> {
```

Comment:
I could easily return another type here besides `PathBuf`.  is string or something else preferable

Reply:
Also on the subject of return type, I think we should return `Option<something>` here, returning `None` (which turns into JSON `null`) when there is no last opened path

Reply:
Here is the exact result from the curl, so it looks like it will be human readable
```
curl http://0.0.0.0:60000/lastusedkeystore
"/tmp/test_cape_wallet.pOvokKcH0DNA"%  
```